build.gradle

```diff
@@ -289,14 +288,9 @@ allprojects {
     }
   }
 
-  // Normalize Xdoclint behaviour across JDKs (OpenJDK 8 is more lenient than Oracle JDK by default).
   javadoc {
-    options.addStringOption('Xdoclint:all', '-quiet')
-    if (JavaVersion.current() == JavaVersion.VERSION_1_8) {
-      options.addStringOption('Xwerror', '-quiet')
-    } else {
-      options.addStringOption('Xwerror', '-html5')
-    }
+    options.addStringOption('Xdoclint:all,-missing', '-quiet')
```

Comment:
I've tried javadoc on:
openjdk11 - failed, number of error. As we require minimum Java11 I think we may need to support this, but question is should we support javadoc options outside of our CI? I'm not sure on this
openjdk18 - all good
oracle jdk11 - same errors as in openjdk11

From my feelings recent javadoc version is more flexible on converting `<, >, <=, >=` things.
Overall I tend more to ignore JDK11 errors as  it's a support thing and it works on some recent JDKs which we use on our CI.